osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -2,154 +2,180 @@
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Graphics.UserInterface;
 using osu.Game.Beatmaps.Timing;
+using osu.Game.Configuration;
 using osu.Game.Rulesets.Scoring;
 using osu.Game.Screens.Play.Break;
 
 namespace osu.Game.Screens.Play
 {
-    public class BreakOverlay : Container
-    {
-        private const double fade_duration = BreakPeriod.MIN_BREAK_DURATION / 2;
-        private const float remaining_time_container_max_size = 0.3f;
-        private const int vertical_margin = 25;
+  public class BreakOverlay : Container
```

Comment:
what happened to spacing in this file?

Reply:
To be honest I'm not quite sure. I could remake this PR because it's getting a little bit messy and there is a bunch of unnecessary commits